dotCMS/build.gradle

```diff
@@ -181,10 +181,10 @@ dependencies {
     /**** And now the libs we pull in from internal company sources - libs stored in ./plugins, ./bin, ./libs, the starter site, etc. ****/
     compile fileTree("src/main/plugins/com.dotcms.config/build/jar").include('plugin-com.dotcms.config.jar')
 
-    starter group: 'com.dotcms', name: 'starter', version: 'empty_20210315', ext: 'zip'
+    starter group: 'com.dotcms', name: 'starter', version: 'empty_20210405', ext: 'zip'
```

Comment:
A new starter version will be required when `release-21.04` gets merged into `master`